devsetup/scripts/edpm-deploy-instance.sh

```diff
@@ -59,10 +59,10 @@ openstack network agent list
 # Create an instance
 openstack server show test || {
     openstack server create --flavor m1.small --image cirros --nic net-id=private test --security-group basic --wait
-    openstack floating ip show 192.168.122.20 || openstack floating ip create public --floating-ip-address 192.168.122.20
-    openstack server add floating ip test 192.168.122.20
+    openstack floating ip show 192.168.122.200 || openstack floating ip create public --floating-ip-address 192.168.122.200
```

Comment:
I think we better do:-
floatip=$(openstack floating ip create public -f value -c floating_ip_address)
openstack server add floating ip test $floatip

Reply:
Ack